<launch>

	<rosparam command="load" file="$(find jaco_learning)/config/teleop_inference.yaml" />

	<param name="setup/sim_mode" value="pybullet" />

	<!-- Trajopt PID control with... -->
	<node name="teleop_inference" pkg="jaco_learning" type="teleop_inference.py" output="screen"/>

</launch>
